### PR TITLE
136 enhance voting experience

### LINKED ISSUE #136: Enhance voting experience
There are several mini issues that may be fixed to really achieve a smooth voting experience:

- [x] #149
- [x] #150
- [x] #151
- [x] #152
- [x] #153
- [x] #154
- [x] #145

### PR DESCRIPTION
**Pull request by Viterbo**
# Description
This is a bunch of fixes to enhance the voting experience

Fixes #136

### CODE REVIEW

**Comment by Viterbo** [comment]
what Issues where fixed in what commit:
- #149
  in [b7a0b26](https://github.com/telosnetwork/app-telos-native/pull/148/commits/b7a0b26e1f2fcfec7c55f433596b0df5891147b4)

- #150
  in [5c25788](https://github.com/telosnetwork/app-telos-native/pull/148/commits/5c25788b1415036c0b7070288f5d8feed70ec131)

- #151
  in [b7a0b26](https://github.com/telosnetwork/app-telos-native/pull/148/commits/b7a0b26e1f2fcfec7c55f433596b0df5891147b4)

- #152
  in [5552fc8](https://github.com/telosnetwork/app-telos-native/pull/148/commits/5552fc89ed3f9d5913caff14084f4257333c4289)

- #153
  in [d359788](https://github.com/telosnetwork/app-telos-native/pull/148/commits/d359788e951a39cc7876ef09cd23c34d669e3af5)

- #154
  in [d0d6a62](https://github.com/telosnetwork/app-telos-native/pull/148/commits/d0d6a62bb926fc0cb1f15dd0f3c5d8d4f470c5a3)

- #145
  in [7880e30](https://github.com/telosnetwork/app-telos-native/pull/148/commits/7880e308652a0c3a605310a4547e14311ee37c1b)

File: src/pages/trails/ballots/view/BallotView.vue [hunk 1]
```diff
@@ -419,7 +425,7 @@ export default {
                             span.statistics-title Request amount
                         div.statistics-body(v-if="ballot.proposal_info")
                             span.text-weight-bold {{ getRequestAmountRounded(ballot.proposal_info.total_requested) }}&nbsp
-            .col-xs-12.col-sm.popup-right-col-wrapper
+            .col-xs-12.col-sm.popup-right-col-wrapper.cosoooooooo
```
Comment (donnyquixotic) [inline src/pages/trails/ballots/view/BallotView.vue:428]: class name (?)  mistake?

**Review by donnyquixotic — CHANGES_REQUESTED** [review]

File: src/pages/trails/ballots/view/BallotView.vue (hunk 1 above)
Comment (Viterbo) [reply src/pages/trails/ballots/view/BallotView.vue]: That would be "thiiiiiiiiing" in spanish. It was a mark I forgot to remove. My bad.
Fixed here [0d2041b](https://github.com/telosnetwork/app-telos-native/commit/0d2041b265080d7a9d821112a953977fa9688ae6)

**Review by karynemayer — APPROVED** [review]

**Comment by Kersten-TCD** [comment]
This looks very good, how is #150 realized, I would envision:

- "voted" tag in the proposal overview

Bildschirmfoto 2022-09-29 um 17 56 53

or

- Dynamic change of the "View proposal and vote" button if the account has voted for this proposal, and show "Voted! View proposal / update vote" (proposed text).

Would make it easy to overview what was voted for.

**Review by donnyquixotic — APPROVED** [review]

**Comment by Viterbo** [comment]
Adding here [e0d0f6f](https://github.com/telosnetwork/app-telos-native/pull/148/commits/e0d0f6f9b1e59a6fb9390fc395efa1cff4761903) the last request from @Kersten-TCD for issue #150

image

**Comment by Viterbo** [comment]
I just rebased this branch with `develop` to be up to date and added a new commit to fix #113 because somehow it was reverted.

**Comment by Viterbo** [comment]
I forgot to add the Voted! sign when displaying ballots in a column. Fixed here [f7df5a3](https://github.com/telosnetwork/app-telos-native/pull/148/commits/f7df5a309165d5598bcd64c7c2bd6ea044cf0fad)

image

**Review by donnyquixotic — APPROVED** [review]

**Review by karynemayer — APPROVED** [review]

**Comment by karynemayer** [comment]
I'm going to merge as is.

@Viterbo I created this #163 with the design for the voted label.

**Pull request merged**